sample-compose/src/main/java/com/airbnb/lottie/sample/compose/player/PlayerPage.kt

```diff
@@ -105,12 +108,8 @@ fun PlayerPage(
         }
     }
 
-    onCommit(outlineMasksAndMattes.value) {
-        animationState.outlineMasksAndMattes = outlineMasksAndMattes.value
-    }
-    onCommit(applyOpacityToLayers.value) {
-        animationState.applyOpacityToLayers = applyOpacityToLayers.value
-    }
+    animationState.outlineMasksAndMattes = outlineMasksAndMattes.value
```

Comment:
I am not exactly sure about this one - I don't think the `onCommit` was needed. Let me know what you think! Maybe this should actually be a `SideEffect`?